src/App.svelte

```diff
@@ -1,96 +1,44 @@
 <script>
 	let visible = false;
-
-	function typewriter(node, { speed = 50 }) {
-		const valid = (
-				node.childNodes.length === 1 &&
-				node.childNodes[0].nodeType === 3
-		);
-
-		if (!valid) {
-			throw new Error(`This transition only works on elements with a single text node child`);
-		}
-
-		const text = node.textContent;
-		const duration = text.length * speed;
-
-		return {
-			duration,
-			tick: t => {
-				const i = ~~(text.length * t);
-				node.textContent = text.slice(0, i);
-			}
-		};
-	}
+	import LeftComponent from './LeftComponent.svelte'
+	import RightComponent from './RightComponent.svelte'
```

Comment:
This is descriptive
But please use more specific naming concepts like it could be Screenshot and Info so it'll easier for you and other developers to understand in the future.